app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -48,6 +46,22 @@ def submission_status
 
     private
 
+    def log_failure(claim)
+      StatsD.increment("#{stats_key}.failure")
+      raise Common::Exceptions::ValidationErrors, claim
+    end
+
+    def log_success(claim)
+      StatsD.increment("#{stats_key}.success")
+      Rails.logger.info "ClaimID=#{claim.confirmation_number} Form=#{claim.class::FORM}"
+    end
+
+    def translate_form4142(form_content)
+      if form_content['form526']['form4142'].present?
```

Comment:
Could we have the conditional outside the method? That way the `translate_form4142` method always translates data and doesn't return nil.

Reply:
I originally had the conditional outside the method, but in order to make sure the variable `form4142` existed when it gets to line 34 I had to have an `else` statement setting `form4142 = nil`. I wasn't able to find an elegant way around that (other than putting the conditional inside the method and allowing it to return nil) but I'm super open to suggestions.

Reply:
only tried this in an irb session but I think this will set `form4142` to nil if present? returns false:
```ruby
form4142 = translate_form4142(form_content) if form_content['form526']['form4142'].present?
```
you could also dig for the hash value:
```ruby
form4142 = translate_form4142(form_content) if form_content.dig('form526', 'form4142')
```